ci/owncloud/opts.py

```diff
@@ -0,0 +1 @@
+../../tests/opts.py
```

Comment:
I'm not sure I like this - why do you need symlink? Why can't you just import where you need opts.py with relative path?

Reply:
See https://github.com/prplfoundation/prplMesh/pull/1302/files#r426802483